content/docs/tech-insights/define-custom-data-sources/index.md

```diff
@@ -122,6 +122,14 @@ If you want to parse YAML content from plain strings, for example from an API En
 * JSONata query: `$parseYaml(spec.definition).openapi`
 * Type: String
 
+**$getEntityData**
+
+If you want to parse request entity information in the JSONata context you can use the `$getEntityData` function. An example query could be like the following:
+* Data Provider Type: Entity Definition
+* Fact Name: OpenAPI Spec Version
+* JSONata query: `$getEntityData('relations')[0].targetRef`
+* Type: JSON Object
```

Comment:
```suggestion
* Type: string
```

Reply:
It returns string, array, object, basically whatever path you use.

Reply:
yup, makes sense